pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -751,6 +752,73 @@ func (k *Bootstrapper) restartPrimaryControlPlane(cfg config.ClusterConfig) erro
 	return nil
 }
 
+func (k *Bootstrapper) SetMinikubeFolderErrorScript(hostDriverIP string) (string, error) {
+	script := fmt.Sprintf(
+		`mkdir c:\var\lib\minikube\certs; Copy-Item C:\etc\kubernetes\pki\ca.crt -Destination C:\var\lib\Minikube\Certs; Remove-Item C:\etc\kubernetes\pki\ca.crt`)
+
+	// Escape double quotes for remote execution
+	script = strings.ReplaceAll(script, `"`, `\"`)
+
+	config := &ssh.ClientConfig{
+		User: "Administrator",
+		Auth: []ssh.AuthMethod{
+			ssh.Password("password"),
```

Comment:
we discussed doing away with password login in favor of ssh keys

Reply:
Fixed!